### PR TITLE
feat(argo-cd): Add labels for argocd-secret

### LINKED ISSUE #1778: ArgoCD secret: add labels field
### Is your feature request related to a problem?

Currently, there is only `configs.secret.annotations` field to configure annotations in `argocd-secret`, but the labels field is missing. There should be the possibility to configure extra labels to the secret too.

### Related helm chart

argo-cd

### Describe the solution you'd like

I'd like to configure additional labels to `argocd-secret` through `configs.secret.labels`

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cgrs**
Resolves #1778

---
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: v2.5.7
 kubeVersion: ">=1.22.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.17.4
+version: 5.18.4
```
Comment (pdrastil) [inline charts/argo-cd/Chart.yaml:6]: Version mismatch

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -23,4 +23,5 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
+    - "[Added]: Extra secret labels with .Values.configs.secret.labels"
     - "[Changed]: Update Argo CD extensions to v0.2.1"
```
Comment (pdrastil) [inline charts/argo-cd/Chart.yaml:27]: This PR is not updating Argo CD extensions

Comment (cgrs) [reply charts/argo-cd/Chart.yaml:27]: Should I add some changelog then?

**Review by pdrastil — APPROVED** [review]
LGTM

**Pull request merged**